pnpm-lock.yaml

```diff
@@ -22264,7 +22264,6 @@ packages:
     resolution: {integrity: sha512-nNPsNE65wjMxEKI93yOP+NPGGBJz/PoN3kZsVLee0XMiJolxSekEVD8wRwBUBqkwc7UWop0edW50yrCQW4CyRw==}
     dependencies:
       undici-types: 5.26.5
-    dev: true
```

Comment:
These are the usually easier to spot questionable differences - where something is no longer dev dep.
cc @jatgarg

Reply:
https://github.com/microsoft/FluidFramework/pull/19746